webapp/package.json

```diff
@@ -50,7 +50,7 @@
     "style-loader": "^3.3.4",
     "ts-jest": "29.1.2",
     "ts-loader": "^9.5.1",
-    "typescript": "^5.4.2",
+    "typescript": "^5.3.3",
```

Comment:
Which lack of support for jest? Maybe I can help!

Reply:
I updated all the dependencies including typescript. It should be solved.